src/site/content/en/blog/link-prefetch/index.md

```diff
@@ -55,6 +61,13 @@ For example, Netflix takes advantage of the time users spend on logged-out pages
 At the time of this writing, it is possible to share prefetched resources among pages served from different origins. When [Double-keyed HTTP cache](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/6KKXv1PqPZ0/oguPntMGDgAJ) ships, this will only work for top-level navigations and same-origin subresources, but it won't be possible to reuse prefetched subresources among different origins. This means that, if `a.com` prefetches the resource `b.com/library.js`, it won't be available in `c.com` cache. Some browsers, such as WebKit-based ones, already [partition caches and HTML5 storage](https://webkit.org/blog/7675/intelligent-tracking-prevention/) for all third-party domains.
 {% endAside %}
 
+The effect of prefetching static assets on performance metrics depends on the resource being prefetched:
+
+- Prefetching web fonts can eliminate layout shifts. In cases where [`font-display: swap;` is used](/font-display/), the swap period for the font is eliminated, resulting in faster rendering of the text and eliminating layout shifts. If a future page uses the prefetched font and the page's LCP element is a block of text using a web font, LCP for that element will also be faster.
```

Comment:
Again I'd probably drop the font example, but maybe it's OK to leave here for those (rare IMHO) occasions where this is the case?